addons/binding/org.openhab.binding.tellstick/README.md

```diff
@@ -1,104 +1,151 @@
+
+***For USB connected tellsticks only, eg. Basic and DUO*** <br>
+*First of all you need to make sure that your JVM is matching your installed Telldus Center. This normally means openHab must run on a 32bit JVM for windows and a 64bit JVM for linux. For windows the binding is hardcoded to look for Telldus Center in Programs Files ("C:/Program Files/Telldus/;C:/Program Files (x86)/Telldus/"). If you have trouble getting the telldus core library to work you can modify the library path using*
 
 ## Thing Configuration
 
-```xtend
-Bridge tellstick:telldus-core:1 "Tellstick Duo" []
+Only the bridges require manual configuration. The devices and sensors should not be added by hand, let the discovery/inbox initially configure these.
 
-Bridge tellstick:telldus-live:2 "Tellstick ZWave" [publicKey="XXX", privateKey="YYYY", token= "ZZZZ", tokenSecret="UUUU"]
-```
+### Dimmers & switches
+There is an option to override the resend count of the commands. Use the option `repeat` for that. Default resend count is 2.
 
-## Channels
+### Bridges
+Depending on your tellstick device type there is different ways of using this binding. The implements two different API:  
+**1)** *Telldus Core* which is a local only interface supported by USB based device. <br>
+**2)** *Telldus Live* which is a REST based cloud service maintained by Telldus. <br>
 
-There are two levels of discovery. When you add this binding it will try to discover the Telldus Core Bridge (this only works if you have Telldus installed and right 32/64 bit version of java).
-If you want to use the Telldus Live bridge, then you manually need to add this thing.
-When you add either of those all devices and sensors will be discovered and reported in the inbox.
+> Not implemented yet but supported by some new devices: <br>
+> **3)** *Local Rest API* is a local API which would work similar to Telldus Live but local.
 
-## Thing Configuration
+Depending on your Tellstick model different API methods is available:
 
-#####Telldus Core Bridge#####
+<table>
+<tr><td><b>Model</b></td> <td><b>Telldus Core</b></td> <td><b>Telldus Live</b></td> <td>Local REST API</td> <td><b>Verified working with openHAB</b></td></tr>
+<tr><td>Tellstick Basic</td><td>X</td><td>X</td><td></td><td></td></tr>
+<tr><td>Tellstick Duo</td><td>X</td><td>X</td><td></td><td>X</td></tr>
+<tr><td>Tellstick Net v.1</td><td></td><td>X</td><td></td><td></td></tr>
+<tr><td>Tellstick Net v.2</td><td></td><td>X</td><td>X</td><td></td></tr>
+<tr><td>Tellstick ZNet Lite v.1</td><td></td><td>X</td><td>X</td><td>X</td></tr>
+<tr><td>Tellstick ZNet Lite v.2</td><td></td><td>X</td><td>X</td><td></td></tr>
+<tr><td>*Tellstick ZNet Pro (Not released)*</td><td></td><td>X</td><td>X</td><td></td></tr>
+</table>
 
-- **libraryPath:** The path to tellduscore.dll/so, 
+#### Telldus Core Bridge
+```
+Bridge tellstick:telldus-core:1 "Tellstick Duo" [resendInterval="200"]
+```
+Optional:
+- **libraryPath:** The path to tellduscore.dll/so,
```

Comment:
please add an empty line before every list as Jekyll otherwise breaks the format